taxonomies/ingredients.txt

```diff
@@ -66193,6 +65719,7 @@ fr:miel non raffiné, miel brut
 en:pure honey
 de:Purer Honig
 fr:Miel pur
```

Comment:
Does it have a parent? I can not see it on the web-interface.

Reply:
The original entries were as follows:
```<en:honey
en:pure honey
de:Purer Honig
fr:Miel pur
```
and
```
<en:honey
fr:miel pur
# ingredient/fr:miel-pur has 3 products @2019-0-28
```